pages/api-reference/customers/create.mdx

```diff
@@ -530,28 +530,28 @@ A verified business Customer must input information on the controller and the Bu
-| passport    | conditional | object | An [optional passport JSON object](/#controller-passport-json-object). **Required** for non-US persons. Includes Passport Identification Number and Country. If [passport](#passport-json-object) is omitted, SSN is required. |
+| Parameter   | Required    | Type   | Description                                                                                                                                                                                                                                                                               |
+| ----------- | ----------- | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| firstName   | yes         | String | The legal first name of the controller.                                                                                                                                                                                                                                                   |
+| lastName    | yes         | String | The legal last name of the controller.                                                                                                                                                                                                                                                    |
+| title       | yes         | String | Job title of the business verified Customer’s controller. IE - Chief Financial Officer                                                                                                                                                                                                    |
+| dateOfBirth | yes         | String | The date of birth of the controller. Formatted in YYYY-MM-DD format. Must be between 18 to 125 years of age.                                                                                                                                                                              |
+| ssn         | conditional | String | Last four or full 9 digits of controller’s Social Security Number. **Required** for US persons. If SSN is omitted, [passport](https://developers.dwolla.com/api-reference/customers/create#controller-passport-json-object) is required.                                                  |
+| address     | yes         | object | An [address JSON object](https://developers.dwolla.com/api-reference/customers/create#controller-address-json-object). Full address of the controller's physical address.                                                                                                                 |
```

Comment:
Same as above, need to change "controller-address-json-object" to an H3 for it to have an anchor.